packages/yew-macro/src/html_tree/html_element.rs

```diff
@@ -346,7 +346,6 @@ impl ToTokens for HtmlElement {
                 }
             }
             TagName::Expr(name) => {
-                #[allow(unused_braces)]
```

Comment:
this is most likely misplaced. It does nothing here. No wonder, cuz the previous paragraphs repeated this pattern, this must have been copy-pasted here.